BlueprintUI/Sources/Layout/Hidden.swift

```diff
@@ -32,6 +32,14 @@ public struct Hidden: Element {
             attributes.isHidden = isHidden
             return attributes
         }
+
+        func sizeThatFits(proposal: SizeConstraint, subelement: LayoutSubelement, cache: inout Cache) -> CGSize {
+            subelement.sizeThatFits(proposal)
+        }
+
+        func placeSubelement(in size: CGSize, subelement: LayoutSubelement, cache: inout ()) {
```

Comment:
This automatically sets the frame of the child, so we don't need to set it here, right?

Reply:
Yeah the default is to fill the parent's frame.